src/System.DirectoryServices/tests/System/DirectoryServices/ActiveDirectory/ActiveDirectoryTests.cs

```diff
@@ -93,6 +93,13 @@ public void TestSchemaProperty()
             }
         }
 
+        [Fact]
```

Comment:
This will throw an exception on the tests machines because they're not domain joined, won't it?

BTW, why not check something like this in? This seems to be a test hole, and this test is better than nothing.

eg  a test like
```c#
string domain;
try {
domain = Domain.GetComputerDomain();
catch (ActiveDirectoryObjectNotFoundException) // not domain joined
{
 return;
}

if (windows)
    Assert.Equal(Environment.GetEnvironmentVariable("USERDOMAIN"), domain);
} else
{
      Assert.NotEmpty(domain);
}

// idempotent
Assert.Equal(domain, Domain.GetComputerDomain());
```

Reply:
I was fiddling around with this to confirm an error in `DirectoryServices`, which I descirbe in https://github.com/dotnet/corefx/issues/35940#issuecomment-493221623. We noticed the issue first in a test failure in Process, which does something similar to what you're describing: https://github.com/dotnet/corefx/blob/02b383221ee03fed74968b7c68a45d626a761339/src/System.Diagnostics.Process/tests/ProcessTests.cs#L1088-L1108

Reply:
That test isn't really a test of GetComputerDomain, but it's not a big deal.